<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0CN418F" ignored="3" name="TicketIT Test(failed)(failed)" tests="1" failures="1" timestamp="31 Oct 2018 10:52:56 GMT" time="14.854" errors="0">
  <testcase name="validateAddedDepartment" time="2.098" classname="com.ticketit.qa.testcase.DepartmentTest">
    <failure type="java.lang.AssertionError" message="expected [HR Department] but found [test]">
      <![CDATA[java.lang.AssertionError: expected [HR Department] but found [test]
	at com.ticketit.qa.testcase.DepartmentTest.validateAddedDepartment(DepartmentTest.java:91)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateAddedDepartment -->
  <testcase name="verifyDepartmentPageLabelTest" time="0.0" classname="com.ticketit.qa.testcase.DepartmentTest">
    <ignored/>
  </testcase> <!-- verifyDepartmentPageLabelTest -->
  <testcase name="validateCreateNewDepartment" time="0.0" classname="com.ticketit.qa.testcase.DepartmentTest">
    <ignored/>
  </testcase> <!-- validateCreateNewDepartment -->
  <testcase name="clickOnDeptTest" time="0.0" classname="com.ticketit.qa.testcase.DepartmentTest">
    <ignored/>
  </testcase> <!-- clickOnDeptTest -->
</testsuite> <!-- TicketIT Test(failed)(failed) -->
